diff --git a/configure.ac b/configure.ac
index 028e4fc78b..ddf638a23b 100644
--- a/configure.ac
+++ b/configure.ac
@@ -550,6 +550,10 @@ then
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		ac_sys_release=`uname -r`
+		;;
 	*-*-vxworks*)
 	    ac_sys_system=VxWorks
 	    ;;
@@ -613,6 +616,9 @@ if test "$cross_compiling" = yes; then
 	*-*-cygwin*)
 		_host_cpu=
 		;;
+	*-*-darwin*)
+		_host_cpu=$host_cpu
+		;;
 	*-*-vxworks*)
 		_host_cpu=$host_cpu
 		;;
diff --git a/configure b/configure
index e8f7981bc5..83072cb277 100755
--- a/configure
+++ b/configure
@@ -4270,6 +4270,10 @@ then
 	*-*-cygwin*)
 		ac_sys_system=Cygwin
 		;;
+	*-*-darwin*)
+		ac_sys_system=Darwin
+		ac_sys_release=`uname -r`
+		;;
 	*-*-vxworks*)
 	    ac_sys_system=VxWorks
 	    ;;
@@ -4335,6 +4338,9 @@ if test "$cross_compiling" = yes; then
 	*-*-cygwin*)
 		_host_cpu=
 		;;
+	*-*-darwin*)
+		_host_cpu=$host_cpu
+		;;
 	*-*-vxworks*)
 		_host_cpu=$host_cpu
 		;;
